test/test_helper.rb

```diff
@@ -0,0 +1,43 @@
+ENV["RAILS_ENV"] = "test"
+require File.expand_path("../../config/environment", __FILE__)
+require "rails/test_help"
+require "minitest/rails"
+require "minitest/reporters" # for Colorized output
+require 'vcr'
+require 'webmock/minitest'
+
+#  For colorful output!
+Minitest::Reporters.use!(
+  Minitest::Reporters::SpecReporter.new,
+  ENV,
+  Minitest.backtrace_filter
+)
+
+VCR.configure do |config|
+  config.cassette_library_dir = 'test/cassettes' # folder where casettes will be located
+  config.hook_into :webmock # tie into this other tool called webmock
+  config.default_cassette_options = {
+    :record => :new_episodes,    # record new data when we don't have it yet
+    :match_requests_on => [:method, :uri, :body] # The http method, URI and body of a request all need to match
+  }
+  # Don't leave our Slack token lying around in a cassette file.
+  config.filter_sensitive_data("<SLACK_TOKEN>") do
```

Comment:
**SLACK_TOKEN**.... **SLACK_TOKEN**... Someone forgot to hide their API keys in the cassettes!